WcaOnRails/db/migrate/20211126105123_update_preferred_formats_again.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class UpdatePreferredFormatsAgain < ActiveRecord::Migration[6.0]
```

Comment:
Maybe instead of Again here, we can name it the regulations date. Possibly like: `RemoveEventFormatsFrom2020Regulations` or something (also not sure what year the regs updated this).